test/mixins/js/ez-tabs.js

```diff
@@ -105,4 +103,38 @@ describe('ez-tabs', function () {
             });
         });
     });
+
+    describe('switch tab in a correct container', function () {
+        let link, label, panel;
+
+        const CLASS_TAB_SELECTED = 'is-tab-selected';
+        const SELECTOR_TAB_SELECTED = '.is-tab-selected';
+
+        element = fixture('AdvancedTestFixture');
+
+        beforeEach(function () {
+            link = element.querySelector('#clickable-link');
+            label = link.parentNode;
+            panel = element.querySelector(link.getAttribute('href'));
+        });
+
+        it('should change tab in a correct tabs container', function () {
```

Comment:
actually here we want to test 2 things so I think you should write 2 tests:
* `should change tab` which should be the same as `'switch tab' / 'should change tab'` (so you should extract the code of it in a function for instance called `testChangeTab` and call it in both cases)
* `should leave others tabs intact` which will also call `testChangeTab` but in addition check that the other tabs have not changed (for that the easiest is probably to compare `.innerHTML` of the other tab container before and after the tab change)